libcontainer/configs/cgroup_unix.go

```diff
@@ -33,6 +37,8 @@ type Cgroup struct {
 	*Resources
 }
 
+const MemoryUnlimited = math.MaxUint64
```

Comment:
This is not the value that is actually used by the kernel, which is 9223372036854771712. Some older kernels will set any value larger than this to 0, and then crash, eg RHEL 7.2.